site/content/en/docs/Integration Patterns/player-capacity.md

```diff
@@ -30,16 +30,15 @@ to 15 players, and if it cannot find one, will allocate a Ready one from the sam
 apiVersion: "allocation.agones.dev/v1"
 kind: GameServerAllocation
 spec:
-  preferred:
-    - gameServerState: Allocated
-      matchLabels:
+  selectors:
+    - matchLabels:
         agones.dev/fleet: lobby
+      gameServerState: Allocated
```

Comment:
I'm curious why you moved `gameServerState:` below `matchLabels:` since I would assume they would naturally go in the order (lexical) order as they were in the old preferred block.

Reply:
In this [document](https://agones.dev/site/docs/reference/gameserverallocation/), All `selectors` order start with `matchLabels:` so I moved to `gameServerState:` below `matchLabels:` .
Do I happen to change order?

Reply:
You changed order in this file from how it was before, but you are right that it is consistent with the reference document (I think the reference doc uses that order to group all alpha fields together, whereas this doc does it in the order that would be output by k8s, which is lexical ordering). Since this is consistent with our existing documentation, I don't think you need to change it.